### PR TITLE
Improve "keyMirror" types & tests

### PR DESCRIPTION
**Pull request by michaeljones**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/STRML/keyMirror (and see the commit message)
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: keymirror/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// Type definitions for keymirror 0.1.1
+// Type definitions for keymirror 0.2.0
+// TypeScript Version: 2.1
```
Comment (andy-ms) [inline keymirror/index.d.ts]: This belongs at the *last* line of the header, after `// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`.

Comment (michaeljones) [inline keymirror/index.d.ts]: Thanks, I'll update this when I get back from work.

**Comment by jfahrenkrug** [comment]
Thanks for these improvements: Could you make sure that the Travis CI tests pass?

**Comment by michaeljones** [comment]
Updated. Seems to be passing now.

**Review by jfahrenkrug — APPROVED** [review]

**Pull request merged**